raylib_opengl_interop.c fails to build with errors like:
```
C:/programming/homedir/raylib/examples/others/raylib_opengl_interop.c: In function 'main':
C:/programming/homedir/raylib/examples/others/raylib_opengl_interop.c:100:5: error: unknown type name 'GLuint'
  100 |     GLuint vao = 0;
      |     ^~~~~~
C:/programming/homedir/raylib/examples/others/raylib_opengl_interop.c:101:5: error: unknown type name 'GLuint'
  101 |     GLuint vbo = 0;
      |     ^~~~~~
C:/programming/homedir/raylib/examples/others/raylib_opengl_interop.c:102:5: error: implicit declaration of function 'glGenVertexArrays' [-Wimplicit-function-declaration]
  102 |     glGenVertexArrays(1, &vao);
      |     ^~~~~~~~~~~~~~~~~
C:/programming/homedir/raylib/examples/others/raylib_opengl_interop.c:103:5: error: implicit declaration of function 'glBindVertexArray'; did you mean 'rlLoadVertexArray'? [-Wimplicit-function-declaration]
  103 |     glBindVertexArray(vao);
      |     ^~~~~~~~~~~~~~~~~
      |     rlLoadVertexArray
```

it should probably be skipped?

Every example also fails to link with a missing symbol from winmm

```
C:\WINDOWS\system32\cmd.exe /C "cd . && C:\programming\toolchains\w64devkit\bin\cc.exe -std=c11 -fno-strict-aliasing -Werror=implicit-function-declaration -Werror=pointer-arith  examples/CMakeFiles/core_clipboard_text.dir/core/core_clipboard_text.c.obj -o examples\core_clipboard_text.exe -Wl,--out-implib,examples\libcore_clipboard_text.dll.a -Wl,--major-image-version,0,--minor-image-version,0  raylib/libraylib.a  -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 && cd ."
C:/programming/toolchains/w64devkit/bin/ld.exe: raylib/libraylib.a(rcore.c.obj):rcore.c:(.text+0x40b0e): undefined reference to `__imp_timeBeginPeriod'
collect2.exe: error: ld returned 1 exit status
```

--- a/cmake/LibraryConfigurations.cmake
+++ b/cmake/LibraryConfigurations.cmake
@@ -187,6 +187,10 @@ elseif ("${PLATFORM}" STREQUAL "Memory")
     set(PLATFORM_CPP "PLATFORM_MEMORY")
     set(GRAPHICS "GRAPHICS_API_OPENGL_SOFTWARE")
     set(OPENGL_VERSION "Software")
+
+    if(WIN32 OR CMAKE_C_COMPILER MATCHES "mingw|mingw32|mingw64")
+        set(LIBS_PRIVATE winmm)
+    endif()
 endif ()
 
 if (NOT ${OPENGL_VERSION} MATCHES "OFF")

--- a/examples/CMakeLists.txt
+++ b/examples/CMakeLists.txt
@@ -105,6 +105,9 @@ elseif ("${PLATFORM}" STREQUAL "DRM")
     list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/rlgl_standalone.c)
     list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/raylib_opengl_interop.c)
 
+elseif ("${PLATFORM}" MATCHES "Memory")
+    list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/raylib_opengl_interop.c)
+
 elseif ("${OPENGL_VERSION}" STREQUAL "Software")
     list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/rlgl_standalone.c)
     list(REMOVE_ITEM example_sources ${CMAKE_CURRENT_SOURCE_DIR}/others/raylib_opengl_interop.c)

